package.json

```diff
@@ -38,7 +38,9 @@
     "shelljs": "0.5.3",
     "underscore.string": "3.2.2",
     "wordwrap": "1.0.0",
+    "yeoman-assert": "2.1.1",
```

Comment:
ok seems like our travis build is failing if its not declared in package so i think its fine